src/main/java/calculator/StringCalculator.java

```diff
@@ -0,0 +1,57 @@
+package calculator;
+
+import java.util.ArrayDeque;
+import java.util.Queue;
+
+public class StringCalculator {
+
+    private Queue<Float> operands = new ArrayDeque<>();
+    private Queue<Operator> operators = new ArrayDeque<>();
+
+    private Float beforeNumber = null;
+
+    public float calculate(String formula) {
+        checkNullOrEmpty(formula);
+
+        String[] formulaSplit = formula.split(" ");
+        separateOperandFromOperator(formulaSplit);
+
+        for(Operator operator = operators.poll() ; operator != null ; operator = operators.poll()) {
+            beforeNumber = calculate(getBeforeNumber(), operator, getAfterNumber());
```

Comment:
이 calculate()는 굳이 분리하지 않아도 될 것 같습니다. 
위에 formula를 처리하는 부분을 따로 분리하면, for 문 만으로 calculate()의 책임이 더 분명해질 것으로 보입니다:)
```suggestion
            beforeNumber = operator.calculate(getBeforeNumber(), getAfterNumber());
```